banzai/main.py

```diff
@@ -91,7 +100,7 @@ def parse_args(extra_console_arguments=None, parser_description='Process LCO dat
                         default=False)
     parser.add_argument('--post-to-elasticsearch', dest='post_to_elasticsearch', action='store_true',
                         default=False)
-    parser.add_argument('--fpack', dest='fpack', action='store_true', default=False,
+    parser.add_argument('--fpack', dest='fpack', action='store_true', default=True,
```

Comment:
I think if you should remove `action='store_true'` or else it will not be possible to set `fpack` to False